lib/modules/versioning/cargo/index.ts

```diff
@@ -149,9 +156,28 @@ function getNewValue({
   return newCargo;
 }
 
+function isBreaking(current: string, version: string): boolean {
+  // The change may be breaking if either version is unstable
+  if (!isStable(version) || !isStable(current)) {
```

Comment:
can this throw somehow? do we validate single version somewhere before?

Reply:
Yes, we only call this if we have a valid update, which implies it was valid versions